File: Contest 4/Sale.java
import java.util.Arrays;
import java.util.Scanner;

public class Sale {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int m = sc.nextInt();
        int a[] = new int[n];

        for (int i = 0; i < n; i++) {
            a[i] = sc.nextInt();
        }
        Arrays.sort(a);
        int sum = 0;
        for (int i = 0; i < m; i++) {
            sum = sum + a[i];
        }
        System.out.println(Math.abs(sum));
    }
}

File: Contest 4/Sale.cpp
#include <iostream>
#include <algorithm>
using namespace std;

int main()
{
    int n, m;
    cin >> n >> m;
    int a[n];

    for (int i = 0; i < n; i++)
    {
        cin >> a[i];
    }

    std::sort(a, a + n);

    int sum = 0;
    for (int i = 0; i < m; i++)
    {
        if (a[i] < 0)
        {
            sum += a[i];
        }
    }

    std::cout << abs(sum) << endl;

    return 0;
}
